### PR TITLE
Ah.filter sites activity counts

### PR DESCRIPTION
**Pull request by huang0h**
## Checklist

- [ ] 1. Change ClickUp ticket status to "In Review"
- [ ] 2. After making the PR, add PR link to ClickUp ticket

## Why

[ClickUp Ticket](https://app.clickup.com/t/xxxxxxx)

### CODE REVIEW

File: service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java [hunk 1]
```diff
@@ -588,6 +594,14 @@ public List<FilterSitesResponse> filterSites(
     if (filterSitesRequest.getNeighborhoodIds() != null)
       filterCondition =
           filterCondition.and(SITES.NEIGHBORHOOD_ID.in(filterSitesRequest.getNeighborhoodIds()));
+      filterCondition = filterCondition.and(SITES.NEIGHBORHOOD_ID.in(filterSitesRequest.getNeighborhoodIds()));
```
Comment (chromium-52) [inline service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java:597]: 🔧 Is this line a duplicate of lines 595 and 596?

**Review by chromium-52 — COMMENTED** [review]
💯 Just one really quick comment but otherwise, lgtm!

File: service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java (hunk 1 above)
Comment (huang0h) [reply service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java]: I thought I made a commit to remove that line 🤦

**Review by chromium-52 — APPROVED** [review]
👍lgtm!!

**Pull request merged**